lib/Tapable.js

```diff
@@ -53,33 +53,37 @@ Tapable.mixin = function mixinTapable(pt) {
 	copyProperties(Tapable.prototype, pt);
 };
 
-Tapable.prototype.applyPlugins = function applyPlugins(name) {
+Tapable.prototype.applyPlugins = function applyPlugins(name, param1, param2) {
 	if(!this._plugins[name]) return;
```

Comment:
Without knowing a lot about this code and how it's being used, I'd suspect that this `this._plugins[name]` is not going to be monomorphic (i.e. `name` will be different and probably `this._plugins` too). So repeating this load is going to be the expensive part.